File: 0.5.4/_static/services/space-center/Vector3.cs
using System;
using System.Net;
using KRPC.Client;
using KRPC.Client.Services.SpaceCenter;

class VectorExample
{
    public static void Main ()
    {
        using (var connection = new Connection ()) {
            var vessel = connection.SpaceCenter ().ActiveVessel;
            Tuple<double,double,double> v = vessel.Flight ().Prograde;
            Console.WriteLine (v.Item1 + ", " + v.Item2 + ", " + v.Item3);
        }
    }
}

File: 0.5.2/_static/services/space-center/Vector3.cpp
#include <iostream>
#include <tuple>
#include <krpc.hpp>
#include <krpc/services/space_center.hpp>

int main() {
  krpc::Client conn = krpc::connect();
  krpc::services::SpaceCenter sc(&conn);
  std::tuple<double, double, double> v = sc.active_vessel().flight().prograde();
  std::cout << std::get<0>(v) << " "
            << std::get<1>(v) << " "
            << std::get<2>(v) << std::endl;
}
